api/v1beta2/ibmvpcmachine_types.go

```diff
@@ -78,6 +78,9 @@ type IBMVPCResourceReference struct {
 	// +kubebuilder:validation:MinLength=1
 	// +optional
 	Name *string `json:"name,omitempty"`
+
+	// IBM Cloud VPC zone
+	Zone *string `json:"vpcZone,omitempty"`
```

Comment:
As we have already a zone field in the `IBMVPCMachineSpec`, this entry may create confusion here

Reply:
Will remove it from here and create new resource VPCResource type with zone.